test/unit/runner/probe/test_run_command_on_host.py

```diff
@@ -7,16 +8,28 @@
 import goth.runner.container.yagna
 from goth.runner.probe import RequestorProbe
 
+CONTAINER_REST_PORT = 6006
 
-@pytest.mark.asyncio
-async def test_run_command_on_host(monkeypatch):
-    """Test if the method `run_command_on_host` works as expected."""
+
+async def env_lines_to_dict(lines):
+    """Convert the lines received from the `env` command into a dictionary."""
+    # The monitor should guarantee that we don't skip any events
+    assert len(lines.past_events) == 0, lines.past_events
+    env = {}
+    async for line in lines:
+        tokens = line.split("=", 1)
+        if len(tokens) == 2:
+            env[tokens[0]] = tokens[1]
+    return env
+
+
+def mock_probe(monkeypatch):
```

Comment:
nitpicking: a more "pytesty" way of doing things here would be to make this function a fixture and include it in the test functions' parameters.

Reply:
converted into a fixture...